src/main/java/org/mule/service/oauth/internal/builder/AbstractOAuthDancerBuilder.java

```diff
@@ -120,6 +121,19 @@ public OAuthDancerBuilder tokenUrl(String tokenUrl, TlsContextFactory tlsContext
     return this;
   }
 
+  @Override
+  public OAuthDancerBuilder<D> tokenUrl(String tokenUrl, TlsContextFactory tlsContextFactory, ProxyConfig proxyConfig) {
```

Comment:
make the other `tokenUrl` and this one call a reusable method to avoid code duplication